src/main/resources/init_db.sql

```diff
@@ -0,0 +1,9 @@
+CREATE DATABASE `taxi_db`;
+
+CREATE TABLE `manufacturers` (
+                                 `id` bigint NOT NULL AUTO_INCREMENT,
+                                 `name` varchar(255) DEFAULT NULL,
```

Comment:
A UNIQUE constraint on the name column in the manufacturers table has not been set.